src/App.jsx

```diff
@@ -1,34 +1,25 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import './App.css'
+import './App.css';
+import React from 'react';
+import { Route, Routes } from 'react-router-dom';
+import Calculator from './components/Calculator';
+import Layout from './pages/Layout';
+import Home from './pages/Home';
+import Quote from './pages/Quote';
 
 function App() {
-  const [count, setCount] = useState(0)
-
   return (
-    <div className="App">
-      <div>
-        <a href="https://vitejs.dev" target="_blank">
-          <img src="/vite.svg" className="logo" alt="Vite logo" />
-        </a>
-        <a href="https://reactjs.org" target="_blank">
-          <img src={reactLogo} className="logo react" alt="React logo" />
-        </a>
-      </div>
-      <h1>Vite + React</h1>
-      <div className="card">
-        <button onClick={() => setCount((count) => count + 1)}>
-          count is {count}
-        </button>
-        <p>
-          Edit <code>src/App.jsx</code> and save to test HMR
-        </p>
-      </div>
-      <p className="read-the-docs">
-        Click on the Vite and React logos to learn more
-      </p>
-    </div>
-  )
+    <>
+      <Routes>
+        <Route path="/" element={<Layout />}>
+          <Route path="/Home" element={<Home />} />
```

Comment:
Well done for implementing the wireframe. Kindly let the home page be the default page to load when launched. Currently here's what shows up on launch. You can do this by adding `index` on the home route e.g `<Route index element={<Home />} />`

![image](https://user-images.githubusercontent.com/20554583/221102691-09b8ab2d-8581-44b2-ab3c-32a041552ddc.png)

Reply:
This change has not yet been implemented well